src/App.js

```diff
@@ -3,28 +3,71 @@ import './App.css';
 import movieData from './data.js';
 import { render } from 'react-dom';
 import Movies from './Movies.js';
+import Form from './Form.js';
+import Preview from './Preview.js'
+import tomato1 from './images/tomato1.svg';
+import tomato2 from './images/tomatos2.svg';
```

Comment:
I added in 4 different tomato icons for us to choose from, we will need to delete these once we pick one